acceptance_modelisation/base_acceptance_map_creator.py

```diff
@@ -338,82 +338,6 @@ def _compute_time_intervals_based_on_fov_rotation(self, obs: Observation) -> Tim
 
         return time_interval
 
-    def _create_base_computation_map(self, observations: Observations) -> Tuple[WcsNDMap, WcsNDMap, WcsNDMap, u.Unit]:
-        """
-        From a list of observations return a stacked finely binned counts and exposure map in camera frame to compute a model
-
-        Parameters
-        ----------
-        observations : gammapy.data.observations.Observations
-            The list of observations
-
-        Returns
-        -------
-        count_map_background : gammapy.map.WcsNDMap
-            The count map
-        exp_map_background : gammapy.map.WcsNDMap
-            The exposure map corrected for exclusion regions
-        exp_map_background_total : gammapy.map.WcsNDMap
-            The exposure map without correction for exclusion regions
-        livetime : astropy.unit.Unit
-            The total exposure time for the model
-        """
```

Comment:
As we have a function with exactly the same signature in both child object, I would leave for now an abstract method with the defined signature. If in the future we break this it would help us to be aware of this point and trigger further check.

Reply:
Ok I will do the change